trailblazer/store/api.py

```diff
@@ -255,6 +255,19 @@ def set_analysis_uploaded(self, case_id: str, uploaded_at: dt.datetime) -> None:
         self.commit()
         LOG.info(f"{analysis_obj.family} - uploaded at set to {uploaded_at}")
 
+    def set_analysis_status(self, case_id: str, status: str):
+        """Setting analysis status to failed."""
+        analysis_obj: models.Analysis = self.get_latest_analysis(case_id=case_id)
+        analysis_obj.status = status
+        self.commit()
+        LOG.info(f"{analysis_obj.family} - Status set to failed")
+
+    def add_comment(self, case_id: str, comment: str):
+        analysis_obj: models.Analysis = self.get_latest_analysis(case_id=case_id)
+        analysis_obj.comment = comment
```

Comment:
I think we should not overwrite any potential existing comments. I prefer appending.
```suggestion
        analysis_obj.comment += " ".join([analysis_obj.comment, comment])
```